src/frontend/packages/marsha-config/tsconfig/tsc-apps.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "App",
+  "extends": "./tsc-base.json",
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "incremental": true,
+    "jsx": "react-jsx",
+    "lib": ["dom", "dom.iterable", "esnext", "es2015"],
+    "module": "esnext",
+    "noFallthroughCasesInSwitch": true,
+    "noEmit": true,
+    "resolveJsonModule": true,
+    "target": "ES2015",
+    "paths": {
+      "@lib-video/*": [
+        "../../../packages/lib_video/src/*",
+        "../../packages/lib_video/src/*"
```

Comment:
I forgot to talk about that, we have to double our path in this way, because we have 2 perspectives the `standalone` one and the `LTI` one, in the lti `tsconfig` the `baseUrl` is at the roots `"."`, while in the standalone the `baseUrl` is at `./src`, so the relative import are different depend the app.